Description: Fix Slow bash completion with AD authentication
 There is hang when using ~ bash-completion with AD
 Add backslash for ~ to specify ~ is special character
Author: Ove Risberg
Origin: other, https://bugs.launchpad.net/ubuntu/+source/bash-completion/+bug/1390061
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/bash-completion/+bug/1390061
Reviewed-by: Seyeong Kim <seyeong.kim@canonical.com>
Last-Update: 2014-11-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/bash_completion
+++ b/bash_completion
@@ -542,9 +542,9 @@
     elif [[ $1 == \'* ]]; then
         # Leave out first character
         printf -v $2 %s "${1:1}"
-    elif [[ $1 == ~* ]]; then
+    elif [[ $1 == \~* ]]; then
         # avoid escaping first ~
-        printf -v $2 ~%q "${1:1}"
+        printf -v $2 \~%q "${1:1}"
     else
         printf -v $2 %q "$1"
     fi
